### ISSUE TITLE
会有一个轻量级的c++版本的hmi吗，dreamview这种形式对于个人开发者来调试自己的内部状态不方便？

### ISSUE THREAD

**Issue by yangmingustb** [ApolloAuto/apollo#15256]
https://github.com/yangmingustb/easy_apollo

基于cmake，我已经开发了一个2d版本的hmi，正在开发3d版本的hmi，来调试决策规划控制。

**Comment by Dstupid** [comment]
旧版Dreamview的交互和使用确实比较麻烦，新版Dreamview+对PnC的调试会友好很多，vehicle viz是3D的，也增强了图表数据的功能，不过目前dv+支持Apollo9.0版本，暂未兼容7.0版本。有升级9.0计划的话可以试用一下~

**Comment by daohu527** [comment]
Yeah, good work, here is a small suggestion: it is best for us to be compatible with the community. If we maintain branches independently, it will not be conducive to the development of this project.

Therefore, I suggest that you organize your plans and merge them into third-party contributions from the community. We are ready to establish a mechanism to complete the contributions. For example, your project is in the incubation stage at the beginning until more people use it and it meets community programming standards. Welcome to discuss together 😃
